### PR TITLE
added Bash Script

### PR DESCRIPTION
**Pull request by YakovAntipenko**
Added Bash Script for Zola -unfinished

### CODE REVIEW

File: zola/zola-bash-script.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+cd ../..
+rm .gitignore
+zola build
+echo "----------DONE----------"
+# Copy build to zola folder
+echo "Copying build to zola folder"
+cp -r ./public/ ../..
+echo "----------DONE----------"
+# Compressing html
+echo "Copying build to zola folder"
```
Comment (Wandalen) [inline zola/zola-bash-script.sh:69]: why do we need zip?

File: .gitignore [hunk 2]
```diff
@@ -13,6 +13,10 @@
 /Cargo.lock
 /.vscode
 /_*
+/zola/public
```
Comment (Wandalen) [inline .gitignore:16]: Add readme with **minimal** instruction and explaining of files tree structure
use command `tree` to get tree of dirs

**Review by Wandalen — CHANGES_REQUESTED** [review]

File: zola/readme.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# MD to HTML file generator (made with Zola)
+
+## What Zola is ?
+
+Zola is static site engine made on Rust
+
+## File tree structure 
+
+.
+├── config.toml
```
Comment (Wandalen) [inline zola/readme.md:10]: right description right here
don't duplicate list of files

File: zola/zola-bash-script.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+# Cloning GitHub Repo with MD files 
+echo "Clonning Repo with MD files"
+cd ../../../
+git clone https://github.com/rust-lang-ua/learn_rust_together.git
+git clone https://github.com/YakovAntipenko/zolafiles.git
+echo "----------DONE----------"
+# Add Zola requirements for MD files
+echo "Add Zola requirements for MD files"
+cd ./learn_rust_together/
+sed -i '1 i\+++' *md
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/zola-bash-script.sh]: updated to single process in last commit

File: zola/makefile [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	bash zola-bash-script.sh 
+
+.PHONY: test
+test:
+	cd ./Zola-blog/hermit_zola/ && zola serve -O 
+
+.PHONY: deploy
+deploy:
+	 cd ../../
+	 curl -H "Content-Type: application/zip" \
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/makefile:18]: example deployment to Netlify 

added as a comment in last commit

File: zola/makefile [hunk 6]
```diff
@@ -0,0 +1,23 @@
+.PHONY: all build test deploy clean
+all: build  deploy test
+
+.PHONY: build
+build: 
+	@echo "Building the html files"
+	bash zola-bash-script.sh 
+
+.PHONY: test
+test:
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/makefile:12]: site test via Zola locally

added as a comment in last commit

File: zola/config.toml [hunk 7]
```diff
@@ -0,0 +1,104 @@
+# Base URL of the site, the only required config argument
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/config.toml]: added in last commit

File: zola/zola-bash-script.sh [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+# Removing ./asset for MD files
+echo "Removing "./asset" from MD files"
+cd ../hermit_zola/content/posts/
+sed -i "s/[.asset][/]//i" *.md 
+sed -i "s/asset//i" *.md
+echo "----------DONE----------"
+# Copying Assets to Zola Static Folder
+echo "Copying Assets to Static"
+cd ../../../learn_rust_together/
+cp -r ./asset/img ../hermit_zola/static/
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/zola-bash-script.sh]: updated with "ln" command in last commit

File: zola/zola-bash-script.sh (hunk 1 above)
Comment (YakovAntipenko) [reply zola/zola-bash-script.sh:80]: zip is one of required format for deployment. 
Netlify for example requires the html`s stored in zip

File: .gitignore (hunk 2 above)
Comment (YakovAntipenko) [reply .gitignore:16]: updated in last commit

File: zola/zola-bash-script.sh [hunk 9]
```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (YakovAntipenko) [reply zola/zola-bash-script.sh:1]: As "Make" is a build automation tool, we got instructions how to build, test, deploy our files.
Our bashfile responsible only for build html files for our specific MD`s.
In that time makefile instructions are universal and could be used with any other build shell-scripts

**Pull request merged**